packages/@webex/providers-cc-widget/.babelrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
Any reason why we have a .babelrc per widget?

Reply:
Babelrc is required for jest testing. Created a base config at root and using that in each workspace, similar to webpack config